cpp/src/gandiva/precompiled/string_ops_test.cc

```diff
@@ -92,6 +96,12 @@ TEST(TestStringOps, TestSpace) {
   EXPECT_EQ(std::string(out, out_len), "    ");
   out = space_int64(ctx_ptr, -5, &out_len);
   EXPECT_EQ(std::string(out, out_len), "");
+  out = space_int64(ctx_ptr, 65536, &out_len);
+  EXPECT_EQ(std::string(out, out_len), std::string(65536, ' '));
+  out = space_int64(ctx_ptr, 9223372036854775807, &out_len);
+  EXPECT_EQ(std::string(out, out_len), std::string(65536, ' '));
+  out = space_int64(ctx_ptr, -2639077559, &out_len);
```

Comment:
I think the error is appearing in the AppVeyor build is because you are using a value that is lower than the minimum integer.

Try to change to `-2639077559L`

Reply:
Seems still not working. But thank you anyway.

Reply:
@projjal can you give us help with that problem? I do not know why the build is still failing even with the tip I gave to @ZMZ91

Reply:
Try `-2639077559LL`. long long is guaranteed to be atleast 64 bit.

Reply:
@ZMZ91 I think Projjal's suggestion worked! I looked at the failed tests and it seems that was caused by environment issues when it tries to install dependencies.

Do the rebase with Arrow's master repository and push it to execute the builds again!

Reply:
Thank you @anthonylouisbsb! BTW, could you help check my another pr https://github.com/apache/arrow/pull/10884? That one was opened even much earlier than this one.